spec/support/schema_matchers.rb

```diff
@@ -4,17 +4,17 @@
 module SchemaMatchers
   extend RSpec::Matchers::DSL
 
-  def valid_against_schema?(json, schema_name)
+  def valid_against_schema?(json, schema_name, opts = {})
     schema_path = Rails.root.join('spec', 'support', 'schemas', "#{schema_name}.json")
-    JSON::Validator.validate!(schema_path.to_s, json, strict: true)
+    JSON::Validator.validate!(schema_path.to_s, json, { strict: true }.merge(opts))
```

Comment:
The EVSS API allows for missing key:value pairs.  `strict: true` basically disallows this so I had to make a way to override it.